.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: hint/needs-triaging, kind/bug
+assignees: ''
+
+---
+
+> Note: For security-related bugs/issues, please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
```

Comment:
won't this be visible in the issue text?

Reply:
<img width="839" alt="image" src="https://user-images.githubusercontent.com/42981373/109377020-7b0fe980-7896-11eb-9dad-5ae836b03301.png">
yes!